runtime/flang/CMakeLists.txt

```diff
@@ -539,20 +565,49 @@ set_source_files_properties(
 # State a dependency on the module
 set_source_files_properties(
   ieee_arithmetic.F95
+  PROPERTIES
+  OBJECT_DEPENDS ${CMAKE_Fortran_MODULE_DIRECTORY}/iso_c_binding.mod
+  OBJECT_OUTPUTS ${CMAKE_Fortran_MODULE_DIRECTORY}/ieee_arithmetic.mod
```

Comment:
You don't need to explicitly add these if you use https://github.com/flang-compiler/flang/pull/464/files#diff-af3b638bc2a3e6c650974192a53c7291R24